tests/test_0_gtk.py

```
@@ -53,24 +53,25 @@
 os.environ['IBUS_TABLE_DEBUG_LEVEL'] = '255'
 
 sys.path.insert(0, "../engine")
-IMPORT_TABLE_SUCCESSFUL = False
+
 try:
     import table
     IMPORT_TABLE_SUCCESSFUL = True
 except (ImportError,):
```

Ruff (0.15.12)

[warning] A length-one tuple literal is redundant in exception handlers

Replace with `except ImportError`

(B013)